packages/cli/src/commands/deploy/service.ts

```diff
@@ -0,0 +1,53 @@
+import { Command, flags } from '@oclif/command'
+import Listr from 'listr'
+import LCD from '@mesg/api/lib/lcd'
+import { compile, create, ICompile, ICreate } from '../../tasks/service'
+import Vault from '@mesg/vault'
+import FileStore from '@mesg/vault/lib/store/file'
+import { join } from 'path'
+import { parse } from 'url'
+import { prompt } from 'inquirer'
+import { Credential } from '../login'
+
+const ipfsClient = require('ipfs-http-client')
+
+export default class Service extends Command {
+  static description = 'Deploy a service'
+
+  static flags = {
+    registry: flags.string({ name: 'Registry to use', required: true, default: 'http://localhost:1317' }),
+    password: flags.string({ description: "Password of your account" }),
+  }
+
+  static args = [{
+    name: 'PATH',
+    description: 'Path or url of a service',
+    default: './'
+  }]
+
+  async run() {
+    const { args, flags } = this.parse(Service)
+    const vault = new Vault<Credential>(new FileStore(join(this.config.configDir, 'credentials.json')))
+    const key = parse(flags.registry).hostname
```

Comment:
a test if the key exists should be added. the current error is:
```
➜  js-sdk git:(feature/deploy-command) ./packages/cli/bin/run deploy:service ../services/service-emit-event-interval
? Type the password of your account [hidden]
Error: localhost is not present
    at Vault.get (~/Development/MESG/js-sdk/packages/vault/lib/index.js:27:19)
    at Service.<anonymous> (~/Development/MESG/js-sdk/packages/cli/src/commands/deploy/service.ts:36:30)
    at Generator.next (<anonymous>)
    at fulfilled (~/Development/MESG/js-sdk/packages/cli/src/commands/deploy/service.ts:5:58)
```

Reply:
added a check to prevent this error with an error that indicates that they need to use the command `login`